File: DataBinding/PageAndRDL/OleDbDataSource/C#/OleDbDataSource/MainForm.cs
using System;
using System.Windows.Forms;
using System.IO;

namespace GrapeCity.ActiveReports.Samples.OleDbDataSource
{
	partial class MainForm : Form
	{
		public MainForm()
		{
			InitializeComponent();
		}

		// Loads and shows the report.
		private void MainForm_Load(object sender, EventArgs e)
		{
			var rptPath = new FileInfo(@"..\..\OleDBReport.rdlx");
			var pageReport = new PageReport(rptPath);
			reportPreview.LoadDocument(pageReport.Document);
		}
	}
}

File: DataBinding/PageAndRDL/OleDbDataSource/VB.NET/OleDbDataSource/MainForm.vb
Imports System.IO

Public Class MainForm

	' Loads and shows the report.
	Private Sub MainForm_Load(ByVal sender As System.Object, ByVal e As EventArgs) Handles MyBase.Load
		Dim rptPath As New FileInfo("..\..\OleDBReport.rdlx")
		Dim pageReport As New PageReport(rptPath)
		reportPreview.LoadDocument(pageReport.Document)
	End Sub

End Class
